docs/getting-started/setup/csp.md

```diff
@@ -18,7 +18,7 @@ CKEditor&nbsp;5 is compatible with applications that use [<abbr title="Content S
 The recommended CSP configuration that allows the rich-text editor to run out–of–the–box with all standard features using the content like images or media from external hosts looks as follows:
 
 ```
-default-src 'none'; connect-src 'self'; script-src 'self'; img-src * data:; style-src 'self' 'unsafe-inline'; frame-src *
```

Comment:
Here, not sure if we should remove all of this, or just the `unsafe-inline` part?  
  
Same below.

Reply:
Just `unsafe-inline` in this one.

`style-src 'self'` should stay in the latter.